### PR TITLE
Add sense

### PR DESCRIPTION
**Pull request by blegat**

### CODE REVIEW

File: src/avg.jl [hunk 1]
```diff
@@ -21,20 +21,10 @@ end
 
 type AvgCutPruner{N, T} <: AbstractCutPruner{N, T}
```
Comment (frapac) [inline src/avg.jl:22]: I like this new interface, its clearer!

File: src/abstract.jl [hunk 2]
```diff
@@ -6,18 +6,59 @@ export CutPruner, AbstractCutPruner, addcuts!, ncuts
...
+| ------- | ------- | ------ |
+| Min     | true    | false  |
+| Max     | true    | true   |
+| ≤       | false   | false  |
+| ≥       | false   | true   |
+
+"""
 abstract AbstractCutPruner{N, T}
 
+function sense2isfunislb(sense::Symbol)
```
Comment (frapac) [inline src/abstract.jl]: I think this name is not that clear. Maybe could we define two functions: 
`getsense(a::Bool, b::Bool)`
`gettype(sense)` 
... if we assume that the tuple of Boolean defines the type of the representation.

What do you think?

File: src/avg.jl (hunk 1 above)
Comment (blegat) [reply src/avg.jl:22]: Thanks ^^ `cuts_DE` meant `D` -> optimality and `E` feasibility (or the opposite I don't remember). Now it does not make sense anymore so `A` is more agnostic :-D

File: src/abstract.jl (hunk 2 above)
Comment (blegat) [reply src/abstract.jl]: That looks better indeed

**Review by frapac — APPROVED** [review]
Ok to merge!

**Pull request merged**